roles/storage/lvm/tasks/install.yml

```diff
@@ -15,6 +15,11 @@
     - ecryptfs-utils 
     - xfsprogs
 
+- name: Storage | Remove existing temporary drive (Azure)
+  sudo: true
+  shell: sudo parted --script /dev/sdb rm 1 || true
+  when: azure
```

Comment:
If this action (removing a drive) could be theoretically applicable to other environments, maybe the name of the variable should be something more generic such as "remove_drive". If it would only ever apply to Azure environments, then it's probably fine.